Description: Fix Bug #777914 about unrecognized identifier
 * moving definition of "extension" from an if loop with condition 
 related to __STDC_VERSION_.
 * I observed that this gets build with gcc-4.9.2, as for gcc 
   __STDC_VERSION__ is not defined.
 * I got these error while using clang to build these.
 * I tested it with clang-3.5(__STDC_VERSION__ 199901L) and
   clang 3.7(__STDC_VERSION__ 201112L) and getting same error in
   Bug #777914. 

Author: Mohit Bhakkad <mohit.bhakkad@gmail.com>
Bug-Debian: https://bugs.debian.org/777914		

--- insserv-1.14.0.orig/listing.h
+++ insserv-1.14.0/listing.h
@@ -47,9 +47,9 @@ typedef unsigned int uint;
 # ifndef  asm
 #  define asm			__asm__
 # endif
-# ifndef  extension
-#  define extension		__extension__
-# endif
+#endif
+#ifndef  extension
+# define extension		__extension__
 #endif
 #ifndef  attribute
 # define attribute(attr)	__attribute__(attr)
